spec/plsql/procedure_spec.rb

```diff
@@ -114,6 +116,32 @@
     end
   end
 
+  describe "BINARY_FLOAT with insufficient precision" do
+
+    before(:all) do
+      plsql.connect! CONNECTION_PARAMS
+      plsql.execute <<-SQL
+      CREATE OR REPLACE FUNCTION test_bfloat( p_a BINARY_FLOAT ) RETURN BOOLEAN
+      IS
+        v_a BINARY_FLOAT := 2.0;
+        v_b BINARY_FLOAT := 3.0;
+      BEGIN
+        RETURN (p_a = (v_a / v_b));
+      END;
+      SQL
+    end
+
+    after(:all) do
+      plsql.execute 'DROP FUNCTION test_bfloat' rescue nil
+      plsql.logoff
+    end
+
+    it "should send a reslut of unprecise division in a way that is matching Oracle math" do
+      expect( plsql.test_bfloat(2.0/3.0) ).to be_truthy
```

Comment:
I would vote for https://github.com/bbatsov/ruby-style-guide as formatting conventions. E.g. - no space after '(', before ')'; and space between `2.0 / 3.0`